drivers/ovsdbDriver.go

```diff
@@ -617,33 +617,33 @@ func (d *OvsdbDriver) GetOfpPortNo(intfName string) (uint32, error) {
 }
 
 // IsVtepPresent checks if VTEP already exists
-func (d *OvsdbDriver) IsVtepPresent(remoteIP string) (bool, string) {
-	d.cacheLock.RLock()
-	defer d.cacheLock.RUnlock()
+func (d *OvsdbDriver) IsVtepPresent(intfName, remoteIP string) bool {
+	condition := libovsdb.NewCondition("name", "==", intfName)
+	selectOp := libovsdb.Operation{
+		Op:    "select",
+		Table: "Interface",
+		Where: []interface{}{condition},
+	}
 
-	// walk the local cache
-	for tName, table := range d.cache {
-		if tName == "Interface" {
-			for _, row := range table {
-				options := row.Fields["options"]
-				switch optMap := options.(type) {
-				case libovsdb.OvsMap:
-					if optMap.GoMap["remote_ip"] == remoteIP {
-						value := row.Fields["name"]
-						switch t := value.(type) {
-						case string:
-							return true, t
-						default:
-							// return false, ""
-						}
-					}
-				default:
-					// return false, ""
+	row, err := d.ovs.Transact(ovsDataBase, selectOp)
```

Comment:
Should we keep the cache look up and fallback to Transact only if there is a miss?